packages/contract-artifacts/artifacts/ZRXToken.json

```diff
@@ -7,221 +7,150 @@
+            },
+            "gasEstimates": {
```

Comment:
We should get rid of these extra fields. Maybe cleaner to have an allowed list and everything else is banned.